modules/ROOT/pages/servicenow/servicenow-connector-studio.adoc

```diff
@@ -0,0 +1,77 @@
+= ServiceNow Studio Configuration - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+
+To configure a connector in Anypoint Studio:
+. Install the connector.
+. Configure the connector.
+. Configure an input source for the connector.
+
+== Install the Connector
+. In Anypoint Studio, click the Anypoint Exchange icon in the Studio task bar.
+. In Anypoint Exchange, click *Login* and supply your Anypoint Platform username and password.
+. Search for the $3 connector, click the card for the connector, and click *Add to project*.
+. Follow the prompts.
+
+== Configure the ServiceNow Connector
+
+. Drag and drop the connector to the Studio Canvas.
+. Configure the Global element for the connector:
++
+// image::servicenow/servicenow-configStudio1.png[servicenow-config-studio]
++
+[%header%autowidth.spread]
+|===
+|Field |Description
+|*Username* | Username to log in with.
+|*Password* | Corresponding password.
+|*Service Address* | WSDL endpoint.
+|*ServiceNow Version* | ServiceNow version.
+|*User tables*| Custom tables for the logged-in user.
+|===
++
+. Go to the Transport Tab and configure a Basic HTTP Transport:
++
+// image::<missing>configStudio2.png[config connector 2]
++
+[%header%autowidth.spread]
+|===
+|Field |Description
+|*Username*| Enter the username to log in with.
+|*Password*| Enter the corresponding password.
+|===
+
+=== Creating an HTTP Global Element for ServiceNow
+
+To create an HTTP global element for the ServiceNow connector, enter values into these fields:
+
+[%header%autowidth.spread]
+|===
+|Field |Description
+|*Protocol*| Protocol selected for the HTTP endpoint, it can be HTTP or HTTPS (secure).
+|*Host*| IP address where the Mule application listens for requests.
+|*Port*| Port address where the Mule application listens for requests.
+|*Base Path*| Path where the Mule application listens for requests.
+|===
+
+== Configure an Input Source
+
+Configure an input source for the connector such as a connector operation,
```

Comment:
Configure an input source for the connector, such as a connector operation, 
(missing comma)